fsck/cmd/getInfo.go

```diff
@@ -168,3 +234,386 @@ func getDataPartitions(addr, name string) ([]*proto.DataPartitionResponse, error
 
 	return dpv.DataPartitions, nil
 }
+
+func getInodePath() (err error) {
+	imap := make(map[uint64]*Inode)
+	if err = buildInodeMap(imap); err != nil {
+		return
+	}
+	if InodeID != 0 {
```

Comment:
queryed all mp of volume by buildInodeMap(), enven if  only show single inode's info.
here may be Optimized later.